test/distributed/org/apache/cassandra/fuzz/topology/HarryTopologyMixupTest.java

```diff
@@ -66,65 +107,130 @@ protected void destroyState(State<Spec> state, @Nullable Throwable cause)
         }
     }
 
-    private static Spec createSchemaSpec(RandomSource rs, Cluster cluster)
+    private static BiFunction<RandomSource, Cluster, Spec> createSchemaSpec(AccordMode mode)
     {
-        ReplayingHistoryBuilder harry = HarryHelper.dataGen(rs.nextLong(),
-                                                            new InJvmSut(cluster),
-                                                            new TokenPlacementModel.SimpleReplicationFactor(3),
-                                                            SystemUnderTest.ConsistencyLevel.QUORUM);
-        cluster.schemaChange(String.format("CREATE KEYSPACE %s WITH replication = {'class': 'SimpleStrategy', 'replication_factor' : 3};", HarryHelper.KEYSPACE));
-        var schema = harry.schema();
-        cluster.schemaChange(schema.compile().cql());
-        waitForCMSToQuiesce(cluster, cluster.get(1));
-        return new Spec(harry);
+        return (rs, cluster) -> {
+            long seed = rs.nextLong();
+            var schema = HarryHelper.schemaSpecBuilder("harry", "tbl").surjection().inflate(seed);
```

Comment:
the table name uses a global static state... and this test runs with a fresh cluster, so I can hard code the table name and avoid non-deterministic fields that impact the test